geometry_msgs/Point[] point_to_apply
bool operation
int64 applier
---
uint8 operation